VideoLocker/src/main/java/org/edx/mobile/util/LocaleUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.edx.mobile.util;
+
+import android.support.annotation.NonNull;
+
+import java.util.Locale;
+
+public class LocaleUtils {
+    public static String getCountryNameFromCode(@NonNull String countryCode) {
+        return new Locale("", countryCode).getDisplayCountry();
```

Comment:
The builder normalized and validated the input; we might want to port that logic here as well…